numpy/ma/core.pyi

```diff
@@ -230,8 +230,8 @@ _ScalarT_co = TypeVar("_ScalarT_co", bound=generic)
 # A subset of `MaskedArray` that can be parametrized w.r.t. `np.generic`
 _MaskedArray: TypeAlias = MaskedArray[_Shape, dtype[_ScalarT]]
 
-MaskType = bool
-nomask: bool
+MaskType: TypeAlias = bool_
+nomask: bool_
```

Comment:
if you wanna be fancy about it, you could also write this as

```suggestion
nomask: bool_[Literal[False]]
```

(assuming that `MaskType` isn't only `np.False_` the case on my machine)

Reply:
fun fact; bitwise operators are implemented in literal `np.bool` during type-checking:
https://github.com/numpy/numpy/blob/bbdfd653f9c158992f5b18586857461629cdb205/numpy/typing/tests/data/reveal/bitwise_ops.pyi#L136-L168